src/ui/marker.js

```diff
@@ -120,8 +126,6 @@ export default class Marker extends Evented {
             this._element = DOM.create('div');
 
             // create default map marker SVG
-            const defaultHeight = 41;
-            const defaultWidth = 27;
```

Comment:
nit: Why move these out? I'd prefer locality, so the values aren't few hundreds of code lines away.

Reply:
My thought was that this makes them less magic-numbery and more consistent with the other constants defined at the start of the file, but that's a good point. Do you think that `ALIGN_TO_HORIZON_BELOW_ZOOM ` and `ALIGN_TO_HORIZON_ABOVE_ZOOM` should also be defined closer to where they are used?

Reply:
>Do you think that ALIGN_TO_HORIZON_BELOW_ZOOM and ALIGN_TO_HORIZON_ABOVE_ZOOM should also be defined closer to where they are used?

I think so, since they're not really shared by anything else and not exported for use outside of this file, it's probably good to keep it local to the block they're used in as well instead of in the global scope. Being consistent in naming would also make sense if you keep here, e.g. making these two variables all caps similar to the other ones.